visualizer.py

```diff
@@ -538,6 +543,8 @@ def show(self, position = "default", back_pointer_location = False):
             else:
                 self.draw.text((10, 50), str(ledsettings.multicolor_range[int(self.currentlocation.replace('Key_range',''))-1][1]), fill = self.text_color)
         
+        if (self.currentlocation == "Shutdown"):
```

Comment:
This part can be moved to a line before
`call("sudo shutdown -h now", shell=True)`
There is a method "render_message" if you need to display an alert before shutting down Pi.